frontend/locales/en/strings.json

```diff
@@ -537,6 +537,7 @@
     "storage": {
       "title": "Shared storage",
       "description": "Configure shared storage for your cluster.",
+      "header": "Storage properties",
```

Comment:
question: why is there a new label for the section title? I would have expected the replacement of `storage.title` with the new phrasing, so that even the old panels get aligned automatically

Reply:
I realize that I have misinterpreted our reference guide, thanks for the catch!
I'm pushing a change to reflect the other PRs approach